Documents/BallotDrafts/sarif-v2.1.0-final-e-ballot-1/sarif-v2.1.0-final.htm

```diff
@@ -8932,7 +8932,7 @@ <h3><a name="_Toc33187409"></a><a name="_Ref526936776">3.17.3 id property</a></h
 
 <p class=MsoNormal><span class=CODEtemp>id</span> <b>MAY</b> consist of a
 single component. This <b>SHALL</b> be taken to specify a unique identifier for
-the run, withough specifying any category that the run belongs to.</p>
+the run, without specifying any category that the run belongs to.</p>
```

Comment:
As is, this isn't a real word, it's a cross between `without`, `though`, and others, `although`. It'd help everyone if a correction is made, `although` it might require some people to confirm what the intended text is.